container.go

```diff
@@ -28,3 +28,7 @@ func (c connectorContainer) Close() error {
 	}
 	return nil
 }
+
+func (c connectorContainer) ID() string {
+	return c.pod.Name
```

Comment:
It may make sense to use the UID instead of the pod name. Especially considering we may deploy multiple containers in a single pod for some use cases.